models/reddit_ads.yml

```diff
@@ -47,14 +47,14 @@ models:
             - campaign_id
             - currency
     columns:
-      - name: date_day
-        description:  '{{ doc("date_day") }}'
+      - name: date_day 
+        description:  '{{ doc("date_day") }}' 
       - name: account_id
         description:  '{{ doc("account_id") }}'
       - name: ad_group_name
         description:  '{{ doc("ad_group_name") }}'
-      - name: ad_group_id
-        description:  '{{ doc("ad_group_id") }}'
+      - name: ad_group_id 
+        description:  '{{ doc("ad_group_id") }}' 
```

Comment:
Should we apply not_null tests to these fields?

Reply:
updated